### ISSUE TITLE
#!/usr/bin/ruby shebang generates false positive for style linter.

### ISSUE THREAD

**Issue by kkirsche** [rubocop/rubocop#1496]
Use of the ruby [shebang](http://en.wikipedia.org/wiki/Shebang_%28Unix%29) `#!/usr/bin/ruby` at the beginning of a Ruby script creates a false positive with the style linter stating:

``` bash
filename.rb:1:1: C: Use snake_case for source file names.
#!/usr/bin/ruby
^
```

Operating System: Mac OS X 10.10.1
Ruby Version: `2.1.2`
Rubocop Version: `0.27.1`

**Comment by zedtux** [comment]
I have this issue too.

```
exe/check-server-accounts:1:1: C: Use snake_case for source file names.
#!/usr/bin/env ruby
^
```

**Comment by philoserf** [comment]
The cop is recommending the file name be `exe/check_server_accounts`. It can be suppressed a few ways. Check the README for those.

**Comment by FunTimeCoding** [comment]
+1

**Comment by danielhz** [comment]
I have this issue too:

```
test.rb:1:1: C: Style/FrozenStringLiteralComment: Missing frozen string literal comment.
#!/usr/bin/env ruby
^
```

(using rubocop 0.81.0)

**Comment by jonas054** [comment]
@danielhz That's not the exact same issue. Run `rubocop -a` to fix the problem. The file will then begin with:
```ruby
#!/usr/bin/env ruby
# frozen_string_literal: true
```

**Comment by danielhz** [comment]
Thank @jonas054! I didn't know about freezing string literals.
